package.json

```diff
@@ -26,7 +26,7 @@
     "metatests": "./bin/cli.js"
   },
   "scripts": {
-    "test": "npm run lint && ./bin/cli.js test",
+    "test": "npm run lint && ./bin/cli.js test --exclude test/mutation",
```

Comment:
Why are these tests excluded?
IFUC these tests are not mutation tests, they are merely tests added after mutation tests to cover untested use cases.

Reply:
Will fix it, I need only to exclude one file there, which exits with code 1.

Reply:
Should I rename it to `unittests`?

Reply:
If you put them under `test/unittests` you should also extract any existing unit tests into that directory. You may also place all of them into `test`

@lundibundi WDYT?